### PR TITLE
Add example with cookies-next package

### PR DESCRIPTION
**Pull request by andreizanik**
Add example with cookies-next package

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thanks, could you convert this to TypeScript to match our [example guidelines](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)? :pray:

Let me know if you need any help!

**Comment by andreizanik** [comment]
> Thanks, could you convert this to TypeScript to match our [example guidelines](https://github.com/vercel/next.js/blob/canary/contributing.md#adding-examples)? 🙏
> 
> Let me know if you need any help!

Done!

File: examples/with-cookies-next/.gitignore [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+yarn-error.log*
+
+# local env files
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+# vercel
+.vercel
```
Comment (balazsorban44) [inline examples/with-cookies-next/.gitignore:34]:
```suggestion
.vercel

# typescript
*.tsbuildinfo
```

File: examples/with-cookies-next/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "start": "next start"
+  },
+  "dependencies": {
+    "cookies-next": "^2.0.5",
+    "next": "latest",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.14",
```
Comment (balazsorban44) [inline examples/with-cookies-next/package.json:15]: `@types/node` should be added as well

File: examples/with-cookies-next/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+## Preview
+
+Preview the example live on [StackBlitz](http://stackblitz.com/):
+
+[![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-cookies-next)
+
+
+## Deploy your own
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
```
Comment (balazsorban44) [inline examples/with-cookies-next/README.md:20]:
```suggestion
## Deploy your own

Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/with-cookies-next)
```

Let's make this consistent with other examples

**Review by balazsorban44 — DISMISSED** [review]
Thanks!

**Comment by andreizanik** [comment]
> Thanks!

There was a problem with automatic checks due to the style code. I fixed it.
I also changed the function name to match the cookie container idiom (as of the latest release of the package).

Please check the pull request again

**Review by balazsorban44 — DISMISSED** [review]

**Comment by andreizanik** [comment]
I get a new problem with auto check 

Build, test, and deploy / lint (pull_request) 
```
lerna notice cli v4.0.0
lerna info ci enabled
lerna ERR! EWORKSPACES Yarn workspaces need to be defined in the root package.json.
lerna ERR! EWORKSPACES See: https://github.com/lerna/lerna/blob/master/commands/bootstrap/README.md#--use-workspaces
 ELIFECYCLE  Command failed with exit code 1.
ERROR: "lint-typescript" exited with 1.
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

I followed the link and realized what I need to add `"workspaces": ["packages/*"]` to package.json 
with this line i get no error locally 

I also found that the prettier finds an error in the `file/examples/cms-tina/.tina/__generated__/schema.gql` 

But this file has a comment
```
# DO NOT MODIFY THIS FILE. This file is automatically generated by Tina
```

After that I did not add any edits, because I do not know how to do it right. I think i need advice

**Review by ijjk — APPROVED** [review]

**Pull request merged**